icu4c/readme.html

```diff
@@ -3,7 +3,7 @@
 
 <html lang="en-US" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-US">
   <head>
-    <title>ReadMe for ICU 68.1</title>
```

Comment:
Thanks Erik!
Would you mind also updating the ICU4J readme file too?
https://github.com/unicode-org/icu/blob/master/icu4j/readme.html

Reply:
Thanks, Erik! That covers the ICU4C Readme version update for me. I created a PR for the ICU4J Readme version update in #1497.

Reply:
Oh thank you very much, Elango!

Reply:
Thanks Elango!